### PR TITLE
chore(db): Make the query more simple to read

### PR DESCRIPTION
**Pull request by solracsf**
## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: lib/private/DB/Adapter.php [hunk 1]
```diff
@@ -46,11 +46,10 @@ public function fixupStatement($statement) {
 	/**
 	 * Create an exclusive read+write lock on a table
 	 *
-	 * @param string $tableName
 	 * @throws Exception
 	 * @since 9.1.0
 	 */
-	public function lockTable($tableName) {
+	public function lockTable(string $tableName) {
```
Comment (susnux) [inline lib/private/DB/Adapter.php:52]: When adding types that we can also add the return type

```suggestion
	public function lockTable(string $tableName): void {
```

Comment (solracsf) [reply lib/private/DB/Adapter.php:52]: That gave me a psalm error... 😞 See initail commit at https://github.com/nextcloud/server/pull/47675/commits/dfcf3132dda2c74ca96a3972ae8d78b4eba55b88

Comment (susnux) [reply lib/private/DB/Adapter.php:52]: Ah yes, because all the implementing classes (AdapterSqlite etc) also would need that typing.

**Review by Altahrim — APPROVED** [review]
I would prefer to get rid of `insertIfNotExist` since it's deprecated since Nextcloud 15 :p 
This function is unsafe since column names are not escaped

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**